lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -53,8 +60,23 @@ def get(self, request, course_key_string):
         course_key = CourseKey.from_string(course_key_string)
         course_tools = CourseToolsPluginManager.get_enabled_course_tools(request, course_key)
 
+        course = get_course_with_access(request.user, 'load', course_key, check_if_enrolled=False)
+        blocks = get_course_date_blocks(course, request.user, request, num_assignments=1)
+
+        # User locale settings
+        user_timezone_locale = user_timezone_locale_prefs(request)
+        user_timezone = user_timezone_locale['user_timezone']
+
+        dates_tab_link = reverse('dates', args=[course.id])
```

Comment:
This reverse here, does it return a relative link or an absolute link? We want to make sure we return absolute links to the frontend.

Relative links look like `/courses/{course_key}/dates` vs an Absolute link that will look like `localhost:18000/courses/{course_key}/dates`